package Exception_handling;

public class test1 {

	public static void main(String[] args) {
		test2.main(null);

		System.out.println("hello");
	}

}
